tests/Resolver/TaxType/CanadaTaxTypeResolverTest.php

```diff
@@ -156,30 +197,56 @@ public function testResolver($resolver)
         $novaScotiaAddress->expects($this->any())
             ->method('getAdministrativeArea')
             ->will($this->returnValue('CA-NS'));
+        $manitobaAddress = $this
+            ->getMockBuilder('CommerceGuys\Addressing\Model\Address')
+            ->getMock();
+        $manitobaAddress->expects($this->any())
+            ->method('getCountryCode')
+            ->will($this->returnValue('CA'));
+        $manitobaAddress->expects($this->any())
+            ->method('getAdministrativeArea')
+            ->will($this->returnValue('CA-MB'));
 
         // Nova Scotia store, Ontario customer.
         $context = $this->getContext($ontarioAddress, $novaScotiaAddress);
         $results = $resolver->resolve($taxable, $context);
-        $result = reset($results);
+        $result  = reset($results);
         $this->assertInstanceOf('CommerceGuys\Tax\Model\TaxType', $result);
         $this->assertEquals('ca_on_hst', $result->getId());
```

Comment:
The tax depends on where the place of supply is deemed to be (http://www.cra-arc.gc.ca/placeofsupply/). As a first approximation, using the purchasers address is reasonable. But a ticket for a conference or other event in a different province than store or customer would be taxed on the venue's location (http://www.cra-arc.gc.ca/tx/bsnss/tpcs/gst-tps/gnrl/hst-tvh/srvcs/lctn-spcfc-vnt-eng.html). For multi-payment recurring purchases, like a membership, the tax is determined by the deemed place of supply at the beginning of each period. So if a monthly music streaming service's customer moved provinces, the following month their tax should change to the new province.